### DISCUSSION TITLE
YALMIP with GPU

### DISCUSSION THREAD

**Discussion by antmw1361** [yalmip/YALMIP#942] [Optimization modelling] [answered]
Hi Johan,

gpuArray command by Matlab seems to be incompatible with sdpvar.  
Is it possible amend the code and sdpsettings to use GPU for optimisation? 

Also, assuming problem cost and constraints can be calculated with gpu enabled functions, how much acceleration should we expect if GPUs are used?

Thanks for your attention.

Kind regards,
Morteza

**Comment by johanlofberg** [comment, marked as answer]
gpu stuff is typically of use when you have massively parallel simple operations on data blocks, completely the opposite of general code which you have here. Hence no support and I see no reason for it to give any benefit

**Reply by antmw1361** [reply]
Thanks for the reply :)
Noted.
